include/openssl/x509_vfy.h

```diff
@@ -184,6 +184,10 @@ void X509_STORE_CTX_set_depth(X509_STORE_CTX *ctx, int depth);
 # define         X509_V_ERR_OCSP_VERIFY_NEEDED                   73  /* Need OCSP verification */
 # define         X509_V_ERR_OCSP_VERIFY_FAILED                   74  /* Couldn't verify cert through OCSP */
 # define         X509_V_ERR_OCSP_CERT_UNKNOWN                    75  /* Certificate wasn't recognized by the OCSP responder */
+/* More 'informational' when looking for issuer cert */
+# define         X509_V_ERR_SIGNATURE_ALGORITHM_MISMATCH         76
+/* When the issuer certificate doesn't have a public key */
+# define         X509_V_ERR_NO_ISSUER_PUBLIC_KEY                 77
```

Comment:
Why did you choose a different alignment? The `X509_V_ERR_OCSP_CERT_UNKNOWN` line already has 127 columns, so it does not make sense to insist on the 80 column rule here, unless you reformat all lines. But the way the existing comments are is more grep friendly, which is IMHO a legal reason for ignoring the 80 column rule here.

Reply:
I didn't pay attention to the previous ones.  Will fix